src/main/java/sim/coffee/Logger.java

```diff
@@ -4,20 +4,20 @@
 import java.io.IOException;
 import java.time.LocalDate;
 
-public class Log {
-	private static Log instance;
+public class Logger {
+	private static Logger instance;
 	private StringBuilder log;
-	
-	private Log() {
+
+	private Logger() {
 		log = new StringBuilder();
 	}
 
-	
+
 	//adds an entry in the log when an order is added to queue (enter), removed from queue (exit) or processed (processed)
 	public void add(Order[] o, String state) {
 		LocalDate time = LocalDate.now();
 		log.append(String.format("%-15s", time));
-		
+
 		switch (state.toLowerCase()) {
 		case "enter":
```

Comment:
Not important but it would probably be good practice to use enums for this kind of scenario (predefined set of options) to enforce correct use of the logger (enums can be defined within a class for proper encapsulation)